docs/quickstart.md

```diff
@@ -174,6 +176,152 @@ sudo tee /etc/containerd/firecracker-runtime.json <<EOF
 EOF
 ```
 
+A similar script for rpm based linux distro e.g. Amazon Linux 2 can be found here
+
+<details>
+
+```
+#!/bin/bash
+
+cd ~
+
+# Install git, Go 1.13, make, curl
+sudo yum -y update
+sudo yum -y install \
+  golang \
+  make \
+  git \
+  curl \
+  e2fsprogs \
+  util-linux \
+  bc \
+  gnupg
+
+
+# Amazon Linux 2 installs go binary is installed in /usr/bin. So no PATH changes required.
+
+cd ~
+
+# Install Docker CE
+# Docker CE includes containerd, but we need a separate containerd binary, built
+# in a later step
+sudo yum -y update
+sudo amazon-linux-extras install -y  docker
+sudo usermod -aG docker $(whoami)
+
+cd ~
+
+# Check out firecracker-containerd and build it.  This includes:
+# * firecracker-containerd runtime, a containerd v2 runtime
+# * firecracker-containerd agent, an inside-VM component
+# * runc, to run containers inside the VM
+# * a Debian-based root filesystem configured as read-only with a read-write
+#   overlay
+# * firecracker-containerd, an alternative containerd binary that includes the
+#   firecracker VM lifecycle plugin and API
+# * tc-redirect-tap and other CNI dependencies that enable VMs to start with
+#   access to networks available on the host
+git clone https://github.com/firecracker-microvm/firecracker-containerd.git
+cd firecracker-containerd
+sudo yum -y install device-mapper
+sg docker -c 'make all image firecracker'
+sudo make install install-firecracker demo-network
+
+cd ~
+
+# Download kernel
+curl -fsSL -o hello-vmlinux.bin https://s3.amazonaws.com/spec.ccfc.min/img/hello/kernel/hello-vmlinux.bin
+
+# Configure our firecracker-containerd binary to use our new snapshotter and
+# separate storage from the default containerd binary
+sudo mkdir -p /etc/firecracker-containerd
+sudo mkdir -p /var/lib/firecracker-containerd/containerd
+# Create the shim base directory for which firecracker-containerd will run the
+# shim from
+sudo mkdir -p /var/lib/firecracker-containerd
+sudo tee /etc/firecracker-containerd/config.toml <<EOF
+disabled_plugins = ["cri"]
+root = "/var/lib/firecracker-containerd/containerd"
+state = "/run/firecracker-containerd"
+[grpc]
+  address = "/run/firecracker-containerd/containerd.sock"
+[plugins]
+  [plugins.devmapper]
+    pool_name = "fc-dev-thinpool"
+    base_image_size = "10GB"
+    root_path = "/var/lib/firecracker-containerd/snapshotter/devmapper"
+
+[debug]
+  level = "debug"
+EOF
```

Comment:
Are these steps distro-agnostic? If so, how about having a section for per-distro commands (apt-get vs. yum) and a section for distro-agnostic commands?